third-party/rsutils/include/rsutils/json.h

```diff
@@ -168,11 +170,15 @@ class nested
     bool is_string() const { return exists() && _pj->is_string(); }
 
     // Get the JSON as a value
-    template< class T > T value() { return json::value< T >( get() ); }
+    template< class T > T value() const { return json::value< T >( get() ); }
     // Get the JSON as a value, or a default if not there
-    template < class T > T value( T const & default_value ) { return json::value< T >( get(), default_value ); }
+    template < class T > T default_value( T const & default_value ) const { return json::value< T >( get(), default_value ); }
+    // Get the object, with a default being an empty one
+    nlohmann::json const & default_object() const { return is_object() ? *_pj : empty_json_object; }
```

Comment:
Can json object be nullptr? Can it be safely dereferenced?

Reply:
Not sure I understand, but yes: `is_object()` checks `exists()` and if that's true then `*_pj` is valid.